src/seedu/addressbook/commands/ExitCommand.java

```diff
@@ -11,6 +11,8 @@
             + "Example: " + COMMAND_WORD;
     public static final String MESSAGE_EXIT_ACKNOWEDGEMENT = "Exiting Address Book as requested ...";
 
+    public ExitCommand() { isMutating = false; }
```

Comment:
In the context of polymorphism, when a subclass wants to change the behaviour of a method that is declared in the superclass, the method would be overriden in the subclass.

Overriding the behaviour by changing the `isMutating` variable may lead to unintended/unexpected behaviours. In this case it may not be apparent, but if the superclass were to change the method's behaviour, then all subclasses would unintentionally follow the superclass' behaviour. Take for instance `adjustSalary` with a variable `salaryWeightage`. If we just change `salaryWeightage` without overriding `adjustSalary`, what would happen if in the future, `adjustSalary`'s behaviour in the superclass is changed to no longer take into account `salaryWeightage`?